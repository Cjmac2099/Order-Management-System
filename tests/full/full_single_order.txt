IQU4RJTDVK
66.39
END
