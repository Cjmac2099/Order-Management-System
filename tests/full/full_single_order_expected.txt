Loading catalog...
Enter 'END' to finish order.
SKU> 
SKU matches product Fresh Celery sticks, price per pound: $2.49
Quantity>
SKU> 
Printing order summary:
66.39lbs Fresh Celery sticks: $165.58
